sites/all/modules/hr/hr_datasets/js/hr_datasets.js

```diff
@@ -28,7 +28,12 @@ Drupal.behaviors.hrDatasets = {
       this.count = response.result.count;
       var fields = [];
       _.each(models, function(resource){
-        fields.push({title: resource.title, url: 'https://data.hdx.rwlabs.org/dataset/' + resource.id});
+        var date = new Date(resource.metadata_modified);
+        var month = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"][date.getMonth()];
+        var resource_link = "https://data.hdx.rwlabs.org/dataset/" + resource.id;
+        var link = '<a href="' + resource_link + '">' + resource.title + '</a>';
+        var last_modified =  month + ' ' + date.getDate() + ', ' + date.getFullYear();
```

Comment:
All the transformations above could be done in Drupal and send only the result in Drupal.settings, that would allow you to send this translated and use l() and other Drupal API functions instead of hardcoding the html